### PR TITLE
Resources

### PR DESCRIPTION
**Pull request by hYdos**

### CODE REVIEW

File: src/main/kotlin/me/hydos/rosella/resource/Assimp.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package me.hydos.rosella.resource
+
```
Comment (hYdos) [inline src/main/kotlin/me/hydos/rosella/resource/Assimp.kt]: maybe a better filename?

**Comment by ramidzkh** [comment]
:crab: works now

File: src/main/kotlin/me/hydos/rosella/Pipeline.kt [hunk 2]
```diff
@@ -29,8 +31,8 @@ class Pipeline() {
 		descriptorSetLayout: Long
 	) {
 		stackPush().use {
-			val vertShaderSPIRV: SpirV = compileShaderFile("shaders/base.v.glsl", ShaderType.VERTEX_SHADER)
-			val fragShaderSPIRV: SpirV = compileShaderFile("shaders/base.f.glsl", ShaderType.FRAGMENT_SHADER)
+			val vertShaderSPIRV: SpirV = compileShaderFile(Global.ensureResource(Identifier("rosella", "shaders/base.v.glsl")), ShaderType.VERTEX_SHADER)
+			val fragShaderSPIRV: SpirV = compileShaderFile(Global.ensureResource(Identifier("rosella", "shaders/base.f.glsl")), ShaderType.FRAGMENT_SHADER)
```
Comment (hYdos) [inline src/main/kotlin/me/hydos/rosella/Pipeline.kt]: this method name is conc

**Comment by hYdos** [comment]
Please fix the conflicts before i merge this.

**Review by hYdos — COMMENTED** [review]
👍🏻

**Pull request merged**